internal/cmd/test.go

```diff
@@ -0,0 +1,198 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type GithubTags struct {
+	TagName string         `json:"tag_name"`
+	URL     string         `json:"url"`
+	Assets  []GithubAssets `json:"assets"`
+}
+
+type GithubAssets struct {
+	Name               string `json:"name"`
+	BrowserDownloadURL string `json:"browser_download_url"`
+	URL                string `json:"url"`
+}
+
+type ResourcesConfig struct {
+	TasksConfig     []ResourceConfig `yaml:"tasks"`
+	PipelinesConfig []ResourceConfig `yaml:"pipelines"`
+}
+
+type ResourceConfig struct {
+	Name    string `yaml:"name"`
+	Version string `yaml:"version"`
+}
+
+type CatalogConfig struct {
+	Resources ResourcesConfig `yaml:"resources"`
+}
+
+type Catalog struct {
+	Version string        `yaml:"version"`
+	Catalog CatalogConfig `yaml:"catalog"`
+}
```

Comment:
Instead of duplicating here 
is it possible to use Catalog struct which is present here https://github.com/openshift-pipelines/catalog-cd/blob/main/internal/contract/contract.go#L44 ??